internal/ingress/controller/template/template.go

```diff
@@ -1177,15 +1177,17 @@ func proxySetHeader(loc interface{}) string {
 
 // buildCustomErrorDeps is a utility function returning a struct wrapper with
 // the data required to build the 'CUSTOM_ERRORS' template
-func buildCustomErrorDeps(upstreamName string, errorCodes []int, enableMetrics bool) interface{} {
+func buildCustomErrorDeps(upstreamName string, errorCodes []int, enableMetrics bool, modsecurityEnabled bool) interface{} {
 	return struct {
 		UpstreamName  string
 		ErrorCodes    []int
 		EnableMetrics bool
+		ModsecurityEnabled bool
 	}{
 		UpstreamName:  upstreamName,
 		ErrorCodes:    errorCodes,
 		EnableMetrics: enableMetrics,
+		ModsecurityEnabled: modsecurityEnabled,
```

Comment:
Unformatted for this section

Reply:
@tao12345666333, @iamNoah1  thanks for the hint. I ran `gofmt -s ./internal/ingress/controller/template/template.go` on the main branch as well as on my branch. For both I got the same output with just this difference here (see screenshot). I don't see an error as I used exactly the same line ending and syntax as in the line before. 


![nginx-ingress-difference-gofmt](https://user-images.githubusercontent.com/52607335/152680161-b2e57133-6a7b-4925-b537-83ee351302cf.png)

Reply:
there is no error, you are right. But the linter wants to have it exactly to be formatted like that.